lib/ansible/modules/network/cloudengine/ce_ospf.py

```diff
@@ -459,7 +459,7 @@ def get_ospf_dict(self, process_id):
         # get nexthop info
         ospf_info["nexthops"] = list()
         nexthops = root.findall(
-            "data/ospfv2/ospfv2comm/ospfSites/ospfSite/ProcessTopologys/ProcessTopology/nexthopMTs/nexthopMT")
```

Comment:
To find all netxhops  should not be from root node of which tag name is 'data'.So remove the first tag 'data' and find all chirld node.